README.md

```diff
@@ -1 +1,68 @@
-# laravel-settings
+## Laravel Settings
+Move your env variables to database to change them easily using laravel nova
+## Requirements
+- PHP >= 8.0
+- Laravel >= 8.0
+- Laravel nova license registered on project
+## Installation
+- Require package with composer:
+```bash
+composer require letsgoi/laravel-settings
+```
+
+- Add `Letsgoi\LaravelSettings\SettingsServiceProvider::class`to `app.php`
+````php
+    /*
+     * Package Service Providers...
+     */
+    ...
+    Letsgoi\LaravelSettings\SettingsServiceProvider::class,
+````
+
+- Publish configuration:
+```bash
+php artisan vendor:publish --provider="Letsgoi\LaravelSettings\SettingsServiceProvider" --tag="migrations"
+```
+
+- Add `SettingNovaResource::class` on `NovaServiceProvider.php`
+````php
+    protected function resources(): void
+    {
+        Nova::resources([
+            ...
+            SettingNovaResource::class,
+        ]);
+    }
+````
+- Add `Observable::make(Setting::class, SettingNovaObserver::class)` on `NovaServiceProvider.php`
+```php
+    public function boot() {
+        parent::boot();
+
+        Observable::make(Setting::class, SettingNovaObserver::class);
+    }
+```
+
+## Usage
+To register a new variable on Settings:
+1. Create a migration with the name and key of the variable: 
+   1. The available types are: bool, string, float, array, bool
+
+```php
+    DB::table('settings')->insert([
+            'key' => 'APP_URL',
+            'type' => 'string'
+    ]);
+```
+2. Run migrations: ``php artisan migrate``
+3. Access your nova url and fill the variable with the value
+4. Use variable in your code using:
+```php
+    private SettingRepository $settingRepository;
+    
+    $value = $this->settingRepository->find('key');
+```
```

Comment:
Provide a better use example. ¿How to create the object? ¿Why `$this` and not `$settingRepository->find(...)`?

Reply:
Fixed, copy-paste issues